src/domain/collection/curatedEvents/__tests__/CuratedEvents.test.tsx

```diff
@@ -99,19 +55,43 @@ test('should show all events', async () => {
       name: { fi: event },
     }))
   );
+  const courses = fakeEvents(
+    courseNames.length,
+    courseNames.map((event, index) => ({
+      id: courseIds[index],
+      name: { fi: event },
+    }))
+  );
   const eventsData = events.data as EventFieldsFragment[];
-  const mocks = getMocks(eventsData);
+  const coursesData = courses.data as EventFieldsFragment[];
+  const eventMocks = getMocks(eventsData, eventIds, 'event');
+  const courseMocks = getMocks(coursesData, courseIds, 'course');
 
-  render(<CuratedEventList collection={collection} />, {
-    mocks,
+  render(<CuratedEvents collection={collection} />, {
+    mocks: [...eventMocks, ...courseMocks],
   });
 
   await waitFor(() => {
     expect(screen.queryByTestId('loading-spinner')).not.toBeInTheDocument();
   });
+
+  // Test that both lists are
+
+  const eventsList = within(screen.getByTestId(eventsListTestId));
   eventNames.forEach((eventName) => {
-    expect(screen.getByText(eventName)).toBeInTheDocument();
+    expect(eventsList.getByText(eventName)).toBeInTheDocument();
+  });
+
+  await waitFor(() => {
+    expect(screen.queryByTestId('loading-spinner')).not.toBeInTheDocument();
   });
+
+  if (isFeatureEnabled('EVENTS_HELSINKI_2')) {
```

Comment:
This if-clause is never executed now as .env has `EVENTS_HELSINKI_2` disabled. It's better to set the featureflag on for the test. Here is what is done in Footer.test.tsx:
```
describe('EVENTS_HELSINKI_2 feature flag', () => {
  [true, false].forEach((EVENTS_HELSINKI_2) => {
    beforeEach(() => {
      setFeatureFlags({ EVENTS_HELSINKI_2 });
    });

    afterEach(() => {
      jest.restoreAllMocks();
    });

    it(`does something when EVENTS_HELSINKI_2 feature is ${
      EVENTS_HELSINKI_2 ? 'on' : 'off'
    }`, () => {
      if (  isFeatureEnabled('EVENTS_HELSINKI_2')) {
           //expect somehting is present
      }
      else {
         // expect something is not present
    });
```

Reply:
Ah, true, forgot that